config/parameters.yaml

```diff
@@ -0,0 +1,117 @@
+# Put parameters here that don't need to change on each machine where the app is deployed
+# https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
+parameters:
+    app.most_important_skills_limit: 6 # used for separing data in the planning
+    app.most_important_skills_to_display: [ch_vpsp, ch_vl] # used for displaying a badge near the user name
+    app.slot_interval: '%env(APP_SLOT_INTERVAL)%'
+    app.available_skills:
+        # Caution: the order matters for the planning display
+        ci_bspp:
+            label: CI Réseau BSPP
+            includes: [ci_samu]
+        ci_samu:
+            label: CI Réseau Samu
+            includes: [ci]
+        ch_vpsp:
+            label: Chauffeur VPSP
+            includes: [pse2, ch_vl]
+        ci:
+            label: CI
+            includes: [pse2]
+        pse2:
+            label: PSE2
+            includes: [pse1]
+        pse1:
+            label: PSE1
+            includes: [psc1]
+        ch_vl:
+            label: Chauffeur VL
+            includes: [benevole]
+        infirmier:
+            label: Infirmier.e local.e
+            includes: [benevole]
+        ce_maraude:
+            label: Chef.fe d'équipe maraudes
+            includes: [solidar]
+        solidar:
+            label: Solidar
+            includes: [maraudeur]
+        maraudeur:
+            label: Maraudeur.se
+            includes: [benevole]
+        tsa_coreg:
+            label: TSA / Coreg
+            includes: [pse2]
+        logisticien:
+            label: Logisticien.ne Croix-Rouge
+            includes: [benevole]
+        resp_perm_soc:
+            label: Responsable Permanence Sociale
+            includes: [benevole]
+        anim_fle:
+            label: Animateur.rice de cours de FLE
+            includes: [benevole]
+        psc1:
+            label: PSC1
+            includes: [benevole]
+        benevole:
+            label: Bénévole
+    app.user_properties:
+        - key: vulnerable
+          type: boolean
+          label: Je fais partie des personnes vulnérables
+          columnLabel: Vulnérable
+          help: |
```

Comment:
we can use nl2br in the view, and remove html here ?

Reply:
indeed